package.json

```diff
@@ -8,7 +8,7 @@
     "_build": "npm run _hugo -- -e dev --buildDrafts --buildFuture --baseURL \"${DEPLOY_PRIME_URL:-http://localhost}\"",
     "__check:format": "./scripts/npx-helper.sh prettier",
     "_check:format:any": "npm run __check:format -- --check --ignore-path ''",
-    "_check:format:ja+zh": "npm run _check:format:nowrap -- content/ja content/zh",
+    "_check:format:ja+uk+zh": "npm run _check:format:nowrap -- content/ja content/uk content/zh",
```

Comment:
Oops, also need to fix the script name further below where the script is called. So I might as well just update the name too.